test/functionalTest/cases/1158_v2_entity_already_exists/create_already_exists.test

```diff
@@ -0,0 +1,87 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+CREATE Entity V2 Already exist
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. POST /v2/entities (Room1-Room)
+# 02. POST /v2/entities (Room1-Room)
+#
+
+echo "01. POST /v2/entities (Room1-Room)"
+echo "=================================="
+payload='{
+  "id": "Room1",
+  "type": "Room",
+  "temp": "29"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "02. POST /v2/entities (Room1-Room)"
+echo "=================================="
+payload='{
+  "id": "Room1",
+  "type": "Room",
+  "temp": "29"
```

Comment:
Change 29 to 30 and see what's happend in the (newley added) step 3:

* If the behaviour is the expected, then we will see 29, as the sedond POST shouldn't have been processed.
* If the behaviour is not the expected, then we will see 30, meaning that the second POST was wrongly processed (I bet for this :)